webapp/components/wpt-results.html

```diff
@@ -161,11 +164,15 @@
         </div>
       </template>
 
-      <template is="dom-if" if="{{ !isLatest }}">
-        <info-banner type="info">
-          Showing results for run <b>{{ sha }}</b>.
-          <a slot="small" href='?sha=latest'>View latest run</a></small>
-        </info-banner>
+      <template is="dom-if" if="[[testRuns.length]]">
+        <template is="dom-if" if="[[resultsRangeMessage]]">
+          <info-banner>
+            [[resultsRangeMessage]]
+            <template is="dom-if" if="[[queryBuilder]]">
+              <paper-button id="edit-query" onclick="[[toggleQueryEdit]]" slot="small">Edit</paper-button>
```

Comment:
When expanded, this still shows "EDIT". Clicking it just collapses the view. I think people will quickly realize clicking it doesn't commit the changes, but maybe it could turn into "CANCEL" or an "X" and throw away changes or something to make it more obvious? That's polish so up to you how far to go :)

Reply:
Will do in follow-up.